index.html

```diff
@@ -607,9 +607,19 @@ <h3>Profile Resolution Semantics</h3>
       <h2>Constraints</h2>
       <section id="Document Encoding">
         <h3>Document Encoding</h3>
-        <p>
-          A Document Instance MUST use UTF-8 character encoding as specified in [[UNICODE]].
-        </p>
+        <p>A <a>Document Instance</a> MUST be concretely encoded as a well-formed XML 1.0 [[!xml]] document using the UTF-8
```

Comment:
What does "concretely encoded" means? Can we simply say "... MUST be a well-formed XML 1.0 document ..." ? Also can we get rid of "Document Instance" as discussed in https://github.com/w3c/dapt/issues/57 ?

Reply:
Suggest we handle "Document Instance" in #57 in a separate PR.

To me, "concretely encoded" means "turned into a sequence of bits", but I'd be happy to adopt a better term if you have one. I think it's helpful to use the same language as https://www.w3.org/TR/ttml2/#concrete-encoding to avoid confusion, but not if it actually creates confusion!

Reply:
> Suggest we handle "Document Instance" in https://github.com/w3c/dapt/issues/57 in a separate PR.

Ok

> To me, "concretely encoded" means "turned into a sequence of bits", but I'd be happy to adopt a better term if you have one. I think it's helpful to use the same language as https://www.w3.org/TR/ttml2/#concrete-encoding to avoid confusion, but not if it actually creates confusion!

A quick Google search shows nothing relevant for "XML concrete encoding" or "XML concretely encoded". I would avoid it. Simple saying "A DAPT document MUST be a well-formed XML 1.0 document encoded using UTF-8".

Reply:
I think the distinction we need to allow for is that a document could be either a serialised sequence of bits, in a file system for example, or an in-memory model (DOM). Only the former needs to be encoded in this way. And there's no requirement to serialise a document at all. So we need something a bit different from your suggestion, somehow. Not sure what, yet. Might be worth seeing how others have handled this in other XML flavours.

Reply:
I'm not convinced we need the distinction. The whole spec is about a serialized format IMHO. If there was no serialization, no way to exchange documents, there would be no need for the spec. The in-memory model, if needed, follows the serialization, as usual with XML/DOM. I quickly looked at SVG and couldn't find the concept of "document instance" (used in one place, but not relevant).

Reply:
Hmm, will think about this, let's discuss.

Reply:
Done in dfcc9c7.